test/fail_compilation/traits.d

```diff
@@ -8,15 +8,23 @@ PERMUTE_ARGS:
 /*
 TEST_OUTPUT:
 ---
-fail_compilation/traits.d(100): Error: `getTargetInfo` key `"not_a_target_info"` not supported by this implementation
-fail_compilation/traits.d(101): Error: string expected as argument of __traits `getTargetInfo` instead of `100`
-fail_compilation/traits.d(102): Error: expected 1 arguments for `getTargetInfo` but had 2
-fail_compilation/traits.d(103): Error: expected 1 arguments for `getTargetInfo` but had 0
+test/fail_compilation/traits.d(100): Error: `getTargetInfo` key `"not_a_target_info"` not supported by this implementation
+test/fail_compilation/traits.d(101): Error: string expected as argument of __traits `getTargetInfo` instead of `100`
+test/fail_compilation/traits.d(102): Error: expected 1 arguments for `getTargetInfo` but had 2
+test/fail_compilation/traits.d(103): Error: expected 1 arguments for `getTargetInfo` but had 0
+test/fail_compilation/traits.d(200): Error: undefined identifier imports.nonexistent
+test/fail_compilation/traits.d(201): Error: undefined identifier imports.nonexistent
```

Comment:
remove the leading `test/`s

Reply:
See the [error](https://auto-tester.puremagic.com/show-run.ghtml?projectid=1&runid=3639135&isPull=true)

```
Test fail_compilation/traits.d failed: 
expected:
----
test/fail_compilation/traits.d(100): Error: `getTargetInfo` key `"not_a_target_info"` not supported by this implementation
test/fail_compilation/traits.d(101): Error: string expected as argument of __traits `getTargetInfo` instead of `100`
test/fail_compilation/traits.d(102): Error: expected 1 arguments for `getTargetInfo` but had 2
test/fail_compilation/traits.d(103): Error: expected 1 arguments for `getTargetInfo` but had 0
test/fail_compilation/traits.d(200): Error: undefined identifier imports.nonexistent
test/fail_compilation/traits.d(201): Error: undefined identifier imports.nonexistent
----
actual:
----
fail_compilation/traits.d(100): Error: `getTargetInfo` key `"not_a_target_info"` not supported by this implementation
fail_compilation/traits.d(101): Error: string expected as argument of __traits `getTargetInfo` instead of `100`
fail_compilation/traits.d(102): Error: expected 1 arguments for `getTargetInfo` but had 2
fail_compilation/traits.d(103): Error: expected 1 arguments for `getTargetInfo` but had 0
fail_compilation/traits.d(200): Error: undefined identifier `imports.nonexistent`
fail_compilation/traits.d(201): Error: undefined identifier `imports.nonexistent`
----
```

Reply:
Figured I might need to do something like that. I thought I saw a leading `test` somewhere, but that might have been a totally different place. I keep just flailing about in a trial-and-error loop because the test suite takes something like half an hour to run on my laptop.
Of course, now that I think about it, I really should have just been running those tests manually and copying the output. My brain is *really* not firing on all cylinders today.

Reply:
There is a script somewhere for autoupdating the error output. possibly in `dlang/tools`?

Reply:
Good to know. I'll keep that in mind. Thanks!

Reply:
> . I keep just flailing about in a trial-and-error loop because the test suite takes at least half an hour to run on my laptop.
Of course, now that I think about it, I really should have just been running those tests manually and copying the output. My brain is really not firing on all cylinders today.

You can run a single test by running `./test/run.d <path>`.

Reply:
> There is a script somewhere for autoupdating the error output. possibly in dlang/tools?

It’s a flag for the `./test/run.d`: https://github.com/dlang/dmd/blob/master/test/README.md#automatically-update-the-test_output-segments